amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectorSettings.scala

```diff
@@ -19,37 +22,71 @@ sealed trait AmqpConnectorSettings {
 
 sealed trait AmqpSourceSettings extends AmqpConnectorSettings
 
-final case class NamedQueueSourceSettings(
-    connectionProvider: AmqpConnectionProvider,
-    queue: String,
-    declarations: immutable.Seq[Declaration] = immutable.Seq.empty,
-    noLocal: Boolean = false,
-    exclusive: Boolean = false,
-    ackRequired: Boolean = true,
-    consumerTag: String = "default",
-    arguments: Map[String, AnyRef] = Map.empty
+final class NamedQueueSourceSettings private (
```

Comment:
Is adding `private` at the end of the class declaration a convention that you are following accros the entire alpakka project?

I personally find more readable `private final class NamedQueueSourceSettings (` or even `final private class NamedQueueSourceSettings (` :)

Reply:
Adding `private` before the argument list makes the constructor private, not the class.